server/lease/lessor.go

```diff
@@ -279,11 +279,16 @@ func (le *lessor) SetCheckpointer(cp Checkpointer) {
 }
 
 func (le *lessor) Grant(id LeaseID, ttl int64) (*Lease, error) {
+	defer func(start time.Time) {
+		leaseGrantDurationSeconds.Observe(time.Since(start).Seconds())
```

Comment:
How this differs than grpc latency metrics for "leaseGrant" request?

Reply:
Ack, don't need this in favor of `grpc_server_handling_seconds_bucket{grpc_method="LeaseGrant"...}`. Will update PR.